executor/index_merge_reader.go

```diff
@@ -389,7 +387,7 @@ func (e *IndexMergeReaderExecutor) startIndexMergeTableScanWorker(ctx context.Co
 			finished:       e.finished,
 			buildTblReader: e.buildFinalTableReader,
 			tblPlans:       e.tblPlans,
-			memTracker:     memory.NewTracker(stringutil.MemoizeStr(func() string { return "TableWorker_" + strconv.Itoa(i) }), -1),
+			memTracker:     memory.NewTracker(i, -1),
```

Comment:
Why there become `i`?

Reply:
em... Any suggestions？

Since the label here doesn't use in any other place, so I think any value will be ok here?

Reply:
already use the `LabelForSimpleTask` here.